src/libcommonserver/utility/utility_mac.mm

```diff
@@ -68,6 +68,11 @@ void restartFinderExtension() {
     }
 
     dispatch_after(dispatch_time(DISPATCH_TIME_NOW, (int64_t) (0.5 * NSEC_PER_SEC)), dispatch_get_main_queue(), ^{
```

Comment:
A comment about what this two lines are doing would be nice to have.

Reply:
Done